google-cloud-logging/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<!-- added to resolve breaking changes until JDK will be upgraded to ≥1.8 -->
+<differences>
+    <!-- Added methods to com.google.cloud.logging.Logging interface with implementation in LoggingImpl is always okay -->
+    <difference>
+        <differenceType>7012</differenceType>
+        <className>com/google/cloud/logging/Logging</className>
+        <method>*</method>
+    </difference>
+    <!-- Added methods to com.google.cloud.logging.spi.v2.LoggingRpc interface with implementation in GrpcLoggingRpc is always okay -->
+    <difference>
+        <differenceType>7012</differenceType>
+        <className>com/google/cloud/logging/spi/v2/LoggingRpc</className>
+        <method>*</method>
+    </difference>
```

Comment:
We should not just add a standing ignore for this breaking change. We cannot be making breaking changes to our clients.

The libraries will migrated to Java 8+ next week and then we can implement no-op default implementations on the interfaces so this won't be necessary (an won't be a breaking change).

Reply:
why is this a breaking change?

Reply:
I'd like avoid discussing whether adding a new method to an interface is a breaking change given we aren't discussing Microsoft COM interfaces.
Since this PR does not release a new version of the library, I would like to propose the following:
Once you finish upgrading to 1.8, please open P1 issue to remove the exceptions configuration and assign the issue to me. I will handle it within a day.
It will let me to avoid putting on hold this change and need for additional merges with more recent work that is already ongoing.
WDYT?

Reply:
We wait until next Thursday (August 5th) to see the upgrade to Java 1.8 is rolled in, so we can remove the exclusion configuration and to resolve the cliff plugin error using default implementation.

Reply:
@chingor13 I see similar exclusion list is already used in the library (see [proto-google-cloud-logging-v2/clirr-ignored-differences.xml](https://github.com/googleapis/java-logging/blob/minherz/create-list-logs/proto-google-cloud-logging-v2/clirr-ignored-differences.xml)). Given that the discussed method for resolving clirr errors after moving the repo to Java 8 did not work, would it be possible to get OK for this part?
I am changing the config to provide fine grained exception list on the method level.